tests/model_hub_tests/tf_tests/nightly_models

```diff
@@ -686,4 +686,453 @@ imagenet/efficientnet_v2_imagenet21k_b2/classification,https://www.kaggle.com/mo
 vgg19-block2-conv2-unpooling-decoder,https://www.kaggle.com/models/emilutz/vgg19-unpooling-encoder-decoder/frameworks/tensorFlow2/variations/vgg19-block2-conv2-unpooling-decoder/versions/1,skip,120720 InvalidArgumentError Graph execution error
 faster_rcnn/resnet50_v1_1024x1024,https://www.kaggle.com/models/tensorflow/faster-rcnn-resnet-v1/frameworks/tensorFlow2/variations/faster-rcnn-resnet50-v1-1024x1024/versions/1,skip,128695 Inference results mismatch
 Wiki-words-500-with-normalization,https://www.kaggle.com/models/google/wiki-words/frameworks/tensorFlow2/variations/500-with-normalization/versions/1,skip,120721 AssertionError No signatures for a model
-wiki40b-lm-zh-tw,https://www.kaggle.com/models/google/wiki40b-lm/frameworks/tensorFlow1/variations/zh-tw/versions/1
\ No newline at end of file
+wiki40b-lm-zh-tw,https://www.kaggle.com/models/google/wiki40b-lm/frameworks/tensorFlow1/variations/zh-tw/versions/1
+albert/albert-base-v2,hf_transformers
+albert/albert-large-v2,hf_transformers
+albert/albert-xxlarge-v2,hf_transformers
+google-bert/bert-base-cased,hf_transformers
+google-bert/bert-base-chinese,hf_transformers
+google-bert/bert-base-german-cased,hf_transformers
+google-bert/bert-base-multilingual-cased,hf_transformers
```

Comment:
please split to have two separate files: `nightly_models_tf_hub` and `nightly_models_hf`.
It will be more convenient to manage.

Reply:
Done.